geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStore.scala

```diff
@@ -156,13 +155,12 @@ class AccumuloDataStore(val connector: Connector,
       val sft = SimpleFeatureTypes.createType(name.getURI, attributes)
 
       // back compatible check if user data wasn't encoded with the sft
-      // noinspection ScalaDeprecation
       if (!sft.getUserData.containsKey(SCHEMA_VERSION_KEY)) {
-        metadata.read(typeName, DTGFIELD_KEY).foreach(sft.setDtgField)
+        metadata.read(typeName, "dtgfield").foreach(sft.setDtgField)
         sft.setSchemaVersion(metadata.readRequired(typeName, VERSION_KEY).toInt)
 
         // If no data is written, we default to 'false' in order to support old tables.
-        if (metadata.read(typeName, SHARED_TABLES_KEY).exists(_.toBoolean)) {
+        if (metadata.read(typeName, "tables.sharing").exists(_.toBoolean)) {
```

Comment:
The variables are deprecated now - instead of leaving them in there and marking them so I just copied the string values here, since they're not used anywhere else.